modules/olm-configuring-aws-sts.adoc

```diff
@@ -0,0 +1,213 @@
+
+During installation, the AWS EFS CSI Driver Operator creates a `CredentialsRequest` custom resource (CR) named `openshift-aws-efs-csi-driver` in the `openshift-cloud-credential-operator` namespace.
+endif::[]
+
+[NOTE]
+====
+By default, `ccoctl` creates objects in the directory in which the commands are run. To create the objects in a different directory, use the `--output-dir` flag. This procedure uses `<path_to_ccoctl_output_dir>` to refer to this directory.
+
+Some `ccoctl` commands make AWS API calls to create or modify AWS resources. You can use the `--dry-run` flag to avoid making API calls. Using this flag creates JSON files on the local file system instead. You can review and modify the JSON files and then apply them with the AWS CLI tool using the `--cli-input-json` parameters.
+====
+
+.Prerequisites
+
+* You have an AWS cluster that is configured to use STS.
+
+* You have installed an Operator that can use STS.
+
+* You have access to the OpenShift CLI (`oc`) that matches your {product-title} version.
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Configure the `ccoctl` utility:
+
+.. Obtain the CCO container image from the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ CCO_IMAGE=$(oc adm release info --image-for='cloud-credential-operator')
+----
+
+.. Save the contents of the cluster pull secret as a JSON file by running the following command:
++
+[source,terminal]
+----
+$ oc get secret installation-pull-secrets -n openshift-image-registry \
+  -o jsonpath="{.data.\.dockerconfigjson}" \
+  | base64 -d > pull-secret.json
+----
+
+.. Extract the `ccoctl` binary from the CCO container image within the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ oc image extract $CCO_IMAGE --file="/usr/bin/ccoctl" -a ./pull-secret.json
+----
+
+.. Change the permissions to make `ccoctl` executable by running the following command:
++
+[source,terminal]
+----
+$ chmod 775 ccoctl
+----
+
+.. Optional: Delete the pull secret file by running the following command:
++
+[source,terminal]
+----
+$ rm ./pull-secret.json
+----
+
+ifndef::olm-generic[]
+. Extract the `CredentialsRequest` custom resource (CR) that the Operator generated during installation by running the following command:
++
+[source,terminal]
+----
+$ oc get credentialsrequest \
+  -n <operator_cr_namespace> <operator_cr_name> \
+  -o yaml > <path_to_directory_with_list_of_credentials_requests>/credrequests/<operator_cr_name>.yaml
+----
+where:
++
+--
+** `<operator_cr_namespace>` is the namespace in which the Operator generates the CR.
+** `<operator_cr_name>` is the name of the CR the Operator generates.
+--
+endif::[]
+
+ifdef::olm-generic[]
+. Create and save a `CredentialsRequest` custom resource (CR) YAML file, such as shown in the following example, and then place it in the `credrequests` directory:
```

Comment:
This is the first place we mention the `credrequests` directory for the generic steps so I was thinking something like this to make it clear why we're putting the CR in a directory with a specific name. The next step mentions the `credrequests` directory too so I could be overthinking it :)

```suggestion
. Create and save a `CredentialsRequest` custom resource (CR) YAML file, such as shown in the following example, and then place it in a `credrequests` directory to be processed by `ccoctl` in a subsequent step:
```